### PR TITLE
SDAF remove soft failures

### PR DESCRIPTION
**Pull request by lpalovsky**
This PR removes workarounds for SDAF bugs which were already solved upstream. 
https://github.com/Azure/sap-automation/issues/617 
https://bugzilla.suse.com/show_bug.cgi?id=1226671

- Related ticket: 
https://github.com/Azure/sap-automation/issues/617
https://bugzilla.suse.com/show_bug.cgi?id=1226671

- Verification run:
https://openqaworker15.qa.suse.cz/tests/302978#live

### CODE REVIEW

File: tests/sles4sap/sap_deployment_automation_framework/deploy_hanasr.pm [hunk 1]
```diff
@@ -73,20 +73,6 @@ sub run {
...
-            my @cmd = ('ansible', 'QES_DB',
-                "--private-key=$sdaf_config_root_dir/sshkey",
-                '--inventory=' . get_required_var('SAP_SID') . '_hosts.yaml',
-                '--module-name=shell',
-                '--args="sudo zypper in -y fence-agents-azure-arm"');
-            assert_script_run(join(' ', @cmd));
-        }
-        sdaf_execute_playbook(%{$playbook_options}, sdaf_config_root_dir => $sdaf_config_root_dir);
-    }
-
```
Comment (lilyeyes) [inline tests/sles4sap/sap_deployment_automation_framework/deploy_hanasr.pm:89]: why removed all the playbooks?

Comment (lpalovsky) [reply tests/sles4sap/sap_deployment_automation_framework/deploy_hanasr.pm:89]: Sorry... a blunder from my side. Fixed. Will re-run VRs

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Pull request merged**